commitlint.config.ts

```diff
@@ -167,14 +229,14 @@ module.exports = {
                             if (line.length > bodyMaxLineLength) {
 
                                 // it's a URL
-                                let containsASpace = line.indexOf(" ") >= 0;
+                                let containsSpace = containsASpace(line);
```

Comment:
I'm thinking this would be more readable if the function is called mightBeUrl, and returns the opposite of "contains a space"; this way it's a function that is more aligned with the other two

Reply:
Done!